docs.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: page
+title: Docs
+permalink: /docs/
+---
+
+# User Guide
+[kanidm book](https://kanidm.github.io/kanidm/)
+
+
```

Comment:
We'll need to add an explicit link to the crate docs - https://kanidm.github.io/kanidm/rustdoc/master/kanidm/